app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/LocalFileTransferActivity.java

```diff
@@ -128,6 +129,24 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     wifiDirectManager.startWifiDirectManager(filesForTransfer);
   }
 
+  void checkForPermissions() {
```

Comment:
why is this package visibility?

Reply:
[Corrected](https://github.com/kiwix/kiwix-android/pull/1538/commits/694cbb37183e9682eedf6bc4d7e876baabb2384d#diff-e1d59ae6e15dcb502e125fb6e243d15dR133)